glific/glific-frontend
# Added error boundary for react

## Summary

Added error boundary for react.

Steps to test:

```
Add a constant in any file
const test: any= {}

now try to nested check for that object

if(test.button.show === true)

It will show the error boundary

```
